docs/source/api/apollo-server.mdx

```diff
@@ -81,48 +83,69 @@ For details, see [Resolvers](../data/resolvers/).
 <tr>
 <td>
 
-##### `introspection`
+##### `schema`
 
-`boolean`
+`Object`
 </td>
 
 <td>
 
-If `true`, enables [schema introspection](https://graphql.org/learn/introspection/) by clients. This is required to use tools that figure out your server's schema from talking to it directly, like Apollo Sandbox and GraphQL Playground. (It's not needed for the non-Sandbox version of Apollo Studio Explorer, which uses the schema published in the Studio schema registry instead.)
+An executable GraphQL schema. You usually don't need to provide this value, because Apollo Server generates it from [`typeDefs`](#typedefs) and [`resolvers`](#resolvers).
```

Comment:
It's a little weird to say it this way. `schema` is really just a more flexible approach than what AS typeDefs + resolvers provides. A few reasons why they might use it include:
* subgraph using `buildFederatedSchema`
* they encountered a limitation with the typeDefs + resolvers approach (`makeExecutableSchema` has a number of options which they might find useful that we don't pass through)
* they've taken a code-first approach rather than schema-first
* they need to perform additional transformations on their schema after creating it

Reply:
Great points. I added your list and tweaked the phrasing as follows:

> An executable GraphQL schema. Under the hood, Apollo Server automatically generates this field from [`typeDefs`](#typedefs) and [`resolvers`](#resolvers).
>
>This field is helpful if you are using:
> - [Apollo Federation](https://www.apollographql.com/docs/federation/implementing-services/#generating-a-federated-schema), which uses the `buildFederatedSchema` function to generate its schema 
> - A function like `makeExecutableSchema` function from [`graphql-tools`](https://www.graphql-tools.com/docs/generate-schema) to create your schema.
> - A code-first approach (i.e., instead of a schema-first approach).

Reply:
👍